src/utils/data/index.js

```diff
@@ -63,6 +67,18 @@ const getBar = (percentage) => {
   return bar;
 };
 
+const getPads = (n) => {
+  if (deviceInfo.osName == "Linux") {
+    if (browserInfo.isChrome) {
+      return FIGURE_SPACE.repeat(n);
+    } else if (browserInfo.isFirefox) {
+      return EN_SPACE.repeat(n);
+    }
```

Comment:
```suggestion
    if (browserInfo.isChrome) {
      return EN_SPACE.repeat(n);
    } else if (browserInfo.isFirefox) {
      return FIGURE_SPACE.repeat(n);
    }
```
Could you please change this and see if it works for you? I just tested out this solution and it worked really well for Ubuntu 22.04 with chrome, firefox and brave, as well as for Windows 10 with chrome, firefox and brave too. Just to note it down: I also tested it against an Iphone 14 pro with chrome and an android (not sure the version) with a chrome, so it seems pretty stable.

Reply:
Hi @GuiLeme , I have made the change. Unfortunately I don't have a server to test your suggestion right now. I just change as you suggested, and added a comment that there might be a locale problem, which we can delete once we know there is any more problem. I hope the community will be happy with it!